coss/home/models.py

```diff
@@ -36,3 +36,19 @@ class Meta:
 
     def __str__(self):
         return self.header_text
+
+
+@register_snippet
+class Testimonial(models.Model):
+    quote = RichTextField()
+    person = models.CharField(max_length=100)
+    photo = models.ImageField(uploaded_to='testimonials')
```

Comment:
The more I think about it, the more I feel that we should have a ForeignKey here for the soon to come profiles. Thoughts? If you agree with this, I would suggest to block this PR until we have the profiles merged.